### PR TITLE
Feature/current savings withdrawals=865bn3tcj

### PR DESCRIPTION
**Pull request by theghost1980**

### CODE REVIEW

File: src/interfaces/list-item.interface.ts [hunk 1]
```diff
@@ -14,3 +14,13 @@ export interface LocalAccountListItem {
   label: string;
   value: string;
 }
+
+export interface CurrentWithdrawingListItem {
```
Comment (cedricguillas) [inline src/interfaces/list-item.interface.ts:18]: put it in a savings.utils file

rename to SavingsWithdrawal

File: src/popup/pages/app-container/app-router.component.tsx [hunk 2]
```diff
@@ -77,6 +78,8 @@ const AppRouter = ({
         return <PendingConersionPageComponent />;
       case Screen.SAVINGS_PAGE:
         return <SavingsPageComponent />;
+      case Screen.CURRENT_WITHDRAW_SAVINGS_PAGE:
```
Comment (cedricguillas) [inline src/popup/pages/app-container/app-router.component.tsx:81]: PENDING_SAVINGS_WITHDRAWAL

File: src/utils/savings.utils.ts [hunk 3]
```diff
@@ -74,6 +75,36 @@ const getRequestId = async (username: string) => {
   );
   return Math.max(...savings.map((e: any) => e.request_id), 0) + 1;
 };
+/* istanbul ignore next */
+const getSavingsWitdrawFrom = async (username: string) => {
```
Comment (cedricguillas) [inline src/utils/savings.utils.ts:79]: typo

Comment (cedricguillas) [reply src/utils/savings.utils.ts:79]: getSavingsWithdrawals

File: src/popup/pages/app-container/app-router.component.tsx (hunk 2 above)
Comment (theghost1980) [reply src/popup/pages/app-container/app-router.component.tsx]: Done. Fixed.

File: src/reference-data/screen.enum.ts [hunk 4]
```diff
@@ -33,6 +33,7 @@ export enum Screen {
   CONVERSION_PAGE = 'CONVERSION_PAGE',
   PENDING_CONVERSION_PAGE = 'PENDING_CONVERSION_PAGE',
   SAVINGS_PAGE = 'SAVINGS_PAGE',
+  CURRENT_WITHDRAW_SAVINGS_PAGE = 'current-witdraw-savings-page',
```
[the remark this answers is not in the document]
Comment (theghost1980) [reply src/reference-data/screen.enum.ts]: Done. Fixed.

File: src/interfaces/list-item.interface.ts (hunk 1 above)
Comment (theghost1980) [reply src/interfaces/list-item.interface.ts]: Placed as savings.interface

**Pull request merged**